src/aarch64.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+This is a part of Loops project.
+Distributed under Apache 2 license.
+See https://github.com/vpisarev/loops/LICENSE
+*/
+
+#ifndef __LOOPS_AARCH64_HPP__
+#define __LOOPS_AARCH64_HPP__
+
+#include "composer.hpp"
+#include "backend.hpp"
+#include "common.hpp"
+#include <vector>
+#include <map>
+
+namespace loops
+{
+enum {
+    A64_LDRSW = 0,
```

Comment:
A64 == ARM64? AMD64? Please, use a longer prefix for instructions

Reply:
Done.